test/e2e/frontend/cypress/tests-ee/instances/logs.spec.js

```diff
@@ -0,0 +1,99 @@
+describe('FlowForge - Instance - Logs', () => {
+    let instance
+    function navigateToInstanceLogs (teamName, instanceName) {
+        cy.request('GET', '/api/v1/user/teams')
+            .then((response) => {
+                const team = response.body.teams.find(
+                    (team) => team.name === teamName
+                )
+                return cy.request('GET', `/api/v1/teams/${team.id}/projects`)
+            })
+            .then((response) => {
+                instance = response.body.projects.find(
+                    (app) => app.name === instanceName
+                )
+                cy.visit(`/instance/${instance.id}/logs`)
+                cy.wait('@getInstance')
+            })
+    }
+
+    beforeEach(() => {
+        cy.intercept('GET', '/api/*/projects/*/logs?*', {
+            body: {
+                meta: {},
+                log: []
+            }
+        }).as('getLogsUpdates')
+
+        cy.intercept('GET', '/api/*/projects/*').as('getInstance')
+
+        cy.login('bob', 'bbPassword')
+        cy.home()
+    })
+
+    it('load all logs by default', () => {
+        // mock the API response with well-defined logs
+        cy.intercept('GET', '/api/*/projects/*/logs', {
+            body: {
+                meta: {},
+                log: [
+                    { level: 'info', msg: 'Updated flows', ts: '16874362594840001' }
+                ]
+            }
+        }).as('getLogs')
+
+        navigateToInstanceLogs('BTeam', 'instance-2-1')
+        cy.wait('@getLogs')
+
+        cy.get('[data-el="instance-log-row"]').should('have.length', 1)
+        cy.get('[data-el="select-ha-replica"]').should('not.exist')
+    })
+
+    it('display a marker to indicuate with HA replica the logs are from, if present', () => {
+        // Modify our Instance so that HA is enabled
+        cy.intercept('GET', '/api/*/projects/*', (req) => {
+            req.continue((res) => {
```

Comment:
@Pezmc TIL about `req.continue()` to _modify_ a response in Cypress, rather than mock up a full response, came in real handy here, and flagging in case you'd not seen it before.